agent/structs/acltemplatedpolicy/policies/ce/nomad-client.hcl

```diff
@@ -2,7 +2,7 @@ agent_prefix "" {
   policy = "read"
 }
 node_prefix "" {
-  policy = "read"
+  policy = "write"
```

Comment:
@tristanmorgan Does Nomad register nodes into Consul? I thought it only registered services to existing Consul nodes.

Reply:
this is from https://hashicorp.atlassian.net/browse/NET-10360?focusedCommentId=518527
> since Consul 1.15 Nomad agents require a node:write policy, and not node:read like our documentation suggests.

Reply:
References from the pr [minimal Consul policy for Nomad agents needs node:write](https://github.com/hashicorp/nomad/pull/23802)

Reply:
Updated policy to limit changes to the named node.